### DISCUSSION TITLE
Motion detection timelapse

### DISCUSSION THREAD

**Discussion by MariaDivs** [gopro/labs#733] [Q&A]
Hello! Can anyone help out a complete novice please?
I'm trying to do a time lapse of my friends new gym business being put together. I've set up the GoPro Labs motion detector on repeat so that its not constantly recording while there's no action, see all the settings below. However when I look at the footage I've captured I have hundreds on 1 second videos rather than one timelapse video merging them all... Is this normal or have I done something wrong here! 
Please help so I can fix the issue for tomorrows capture! Thanks so much!!
![IMG_7302](https://github.com/gopro/labs/assets/157536010/807055f5-e280-473f-9854-53a2c4992f12)
![IMG_7298](https://github.com/gopro/labs/assets/157536010/fb5aa857-bfc8-40d1-b112-26f1838034b8)
![IMG_7299](https://github.com/gopro/labs/assets/157536010/4ae7515f-59a2-4cee-8091-3d0a25911ca6)
![IMG_7300](https://github.com/gopro/labs/assets/157536010/8a060afd-9880-4587-aa67-6b6db888521f)
![IMG_7301](https://github.com/gopro/labs/assets/157536010/78d1da52-1154-4f83-9a71-4ede40d8d661)

**Comment by dnewman-gpsw** [comment]
That's a super crazy command. Keep commands below 20 characters until you know what you're doing. You are setting the owner information every time it loops, which is not healthy for the camera. While it is hard to hurt the camera, Labs does not include guardrails. It's designed to get the most power, but at some risk, so start simple  

Motion detection I believe can work with time lapse video. The command should be like !SM4H5!R, super simple.

**Reply by MariaDivs** [reply]
Thanks so much for the feedback! This whole thing has blown my mind and I thought I needed to edit everything in the boxes ha! Will tweak it all now thank you! :-)

**Comment by dnewman-gpsw** [comment]
After returning the office, I tried TLV with !SM4H5!R, but it doesn't work on the latest models 11 & 12. I might work on earlier modes, you did mention which you are using.   An alternative is you used the sound trigger.  It will only timelapse while there is sound. You need voice control active and use a command like this !SD55H9!R (record above 55dB SPL), I have confirmed this will work on HERO12.
